Library/Homebrew/dev-cmd/squash-bottle-pr.rb

```diff
@@ -13,7 +13,11 @@ def squash_bottle_pr
     marker = "Build a bottle for Linuxbrew"
     safe_system "git", "reset", "--hard", "HEAD~2"
     safe_system "git", "merge", "--squash", head
-    safe_system "sed", "-i", "", "-e", "/^#.*: #{marker}$/d", file
+
+    # HACK: achieve compatibility between BSD and GNU sed
+    # by using a dummy backup and then deleting it
+    safe_system "sed", "-iorig", "-e", "/^#.*: #{marker}$/d", file
+    rm_f file.to_s + "orig"
```

Comment:
Note: you have to use `to_s` here explicitly or else you end up with a "/" character between ".rb" and "orig".